codeworld-compiler/codeworld-compiler.cabal

```diff
@@ -54,7 +48,6 @@ Library
     directory,
     exceptions,
     filepath,
-    ghc-lib-parser < 8.8,
```

Comment:
Can you leave this and the stuff in CodeWorld.Compile.Framework?  I already have one other place I'd like to use it (#1073).

Reply:
One of the reasons I got rid of this as soon as possible was that when I try to build with it as is, I get the following error preventing me from building:

[2 of 4] Compiling CodeWorld.Compile.Framework ( src/CodeWorld/Compile/Framework.hs, dist/build/CodeWorld/Compile/Framework.o )

src/CodeWorld/Compile/Framework.hs:65:1: error:
    Could not find module ‘Platform’
    Perhaps you meant Platform (needs flag -package-key ghc-8.6.5)
   |
65 | import qualified "ghc-lib-parser" Platform as GHCParse
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Do you have any idea what's causing this? If so, I can bring it back now; if not, I'll leave it out until I've finished working on integration and then bring it back for you to look at, as otherwise I can't build to test anything.

Reply:
It looks like the `Platform` module was removed (or flag-protected?) between 0.20190603 and 0.20190703.  You can add a tighter version bound to ghc-lib-parser in the cabal file.  Sorry about that!

Reply:
I've fixed this bound and restored the code in Framework.hs.